src/main/java/edu/oregonstate/cs361/battleship/BattleshipModel.java

```diff
@@ -6,24 +6,21 @@
 
 public class BattleshipModel {
 
-
     //                  Member variables
+    protected Ship[] playerArray;
     protected Ship playerAircraftCarrier;
 
+    protected Ship[] computerArray;
     protected Ship computerAircraftCarrier;
 
-    // all ships for player or computer are collected in these lists
```

Comment:
It feels weird swapping out a versatile List for hard coding array indexes for each ship, however the Gson bug is making this the only option.